### PR TITLE
Cannot add organization owner as collaborator to project

### PR DESCRIPTION
**Pull request by why-not-try-calmer**
Evidence that this works:
Capture d’écran du 2023-08-18 08-37-34

Capture d’écran du 2023-08-18 08-37-03

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-2646 Adding the org owner as collaborator gives error 500 using the web UI](https://app.clickup.com/t/2192114/QF-2646)

**Review by suricactus — APPROVED** [review]
Thanks

File: docker-app/qfieldcloud/core/models.py [hunk 1]
```diff
@@ -1478,7 +1478,12 @@ def clean(self) -> None:
             if self.collaborator.is_person:
                 members_qs = organization.members.filter(member=self.collaborator)
 
-                if not members_qs.exists():
+                # for organizations-owned projects, the candidate collaborator
+                # must be a member of the organization or the organization's owner
```
Comment (suricactus) [inline docker-app/qfieldcloud/core/models.py:1482]:
```suggestion
                # for organizations-owned projects, the candidate collaborator
                # must be a member of the organization or and not organization's owner
```

Comment (suricactus) [reply docker-app/qfieldcloud/core/models.py:1482]: @why-not-try-calmer isn't it more correct now?

Comment (why-not-try-calmer) [reply docker-app/qfieldcloud/core/models.py:1482]: As of this PR, the `if` above will **not** lead to a ValidationError if the candidate collaborator is the organization's owner.

Comment (suricactus) [reply docker-app/qfieldcloud/core/models.py:1482]: Ok, I see, I misread the `not` in front of the condition.

**Review by suricactus — APPROVED** [review]

**Pull request merged**